go-controller/extensions/pkg/node/node_windows.go

```diff
@@ -40,4 +135,84 @@ func (n *nodeController) Delete(node *kapi.Node) {
 //  1. Setting up this node and its VxLAN extension for talking to other nodes
 //  2. Setting back annotations about its VTEP and gateway MAC address to its own node object
 func (n *nodeController) InitSelf() {
+	// Create network
+	var networkInfo NetworkInfo
+
+	// Retrieve the host prefix from the annotations
+	hostsubnet, ok := n.thisNode.Annotations[extensionHostSubnet]
+
+	if !ok {
+		log.Errorf("Couldn't retreive the host subnet from the '%s' node's annotations", n.thisNode.Name)
+		return
+	}
+
+	_, addressPrefix, err := net.ParseCIDR(hostsubnet)
+	if err != nil {
+		log.Error(err)
+		return
+	}
+
+	// Retrieve the host prefix from the annotations
+	gatewayIP, ok := n.thisNode.Annotations[extensionDrIp]
```

Comment:
@JocelynBerrendonner On thisNode, what is actually supposed to determine the DRIP? I can't see anywhere that sets this annotation and the master-side doesn't know what it should be, right?

Reply:
@JocelynBerrendonner per call today, you can use pkg/util/net.go NextIP() on a net.ParseCIDR() of the HostSubnet to get a gateway IP if you want.

Reply:
Fixed in the latest iteration!